app/css/main.css

```diff
@@ -127,7 +137,7 @@ main {
   padding-bottom: 20vh;
 }
 
-.hero-title {
+.hero .hero-title {
```

Comment:
I removed the leading `.hero` in most `.hero .hero-*` selectors because it actually isn't needed. Please take a look at the diff and revert if unneeded.

Reply:
I've removed .hero before .hero-title & .hero-caption